libraries/core_libs/network/include/network/tarcap/taraxa_peer.hpp

```diff
@@ -65,6 +65,24 @@ class TaraxaPeer : public boost::noncopyable {
 
   const dev::p2p::NodeID &getId() const { return id_; }
 
+  /**
+   * @brief Reports suspicious pacet
+   *
+   * @return true in case suspicious packet count within current minute is greater than kMaxSuspiciousPacketPerMinute
+   */
+  bool reportSuspiciousPacket() {
+    uint64_t now =
+        std::chrono::duration_cast<std::chrono::minutes>(std::chrono::system_clock::now().time_since_epoch()).count();
+    // Reset counter if no suspicious packet sent for over a minute
+    if (timestamp_suspicious_packet_ > now) {
```

Comment:
how can this be true?

Reply:
Fixed and also I added a unit test in NetworkTest to verify this method is acting as expected